api/application_msgraph.go

```diff
@@ -323,9 +323,11 @@ func (c AppClient) removePasswordResponder(resp *http.Response) (autorest.Respon
 
 func (c AppClient) createApplicationPreparer(ctx context.Context, displayName string) (*http.Request, error) {
 	parameters := struct {
-		DisplayName *string `json:"displayName"`
+		DisplayName    *string `json:"displayName"`
+		SignInAudience *string `json:"signInAudience"`
 	}{
-		DisplayName: to.StringPtr(displayName),
+		DisplayName:    to.StringPtr(displayName),
+		SignInAudience: to.StringPtr("AzureADMyOrg"),
```

Comment:
This is needed to fix an issue with `persist_app` since the default value will only allow 2 active tokens at a time.  

If needed I can pull this into its own PR.